r_markdown/emissions.Rmd

```diff
@@ -196,21 +320,20 @@ ___
 
 * account for artifact
 
+Use replicate average artifact (probably clean up spreadsheet at some point)
```

Comment:
I didn't interpret to the code correctly, you're right, you're right (I thought your were averaging across test replicates not measurement replicates)